lib/go-tc/deliveryservices.go

```diff
@@ -592,3 +592,32 @@ type AssignedDsResponse struct {
 	DSIds    []int `json:"dsIds"`
 	Replace  bool  `json:"replace"`
 }
+
+type DeliveryServiceSafeUpdate struct {
+	DisplayName *string
+	InfoURL     *string
+	LongDesc    *string
+	LongDesc1   *string
```

Comment:
Done.

The tags aren't necessary, because it's only deserialized and encoding/json will fall back to a case-insensitive match. But I agree, explicit is better than implicit.